lms/djangoapps/static_template_view/urls.py

```diff
@@ -1,36 +1,37 @@
+
 """
 URLs for static_template_view app
 """
 
 from django.conf import settings
-from django.conf.urls import patterns, url
+from django.conf.urls import url
 
-urlpatterns = (
-    'static_template_view.views',
+from static_template_view.views import index, render, render_404, render_500, render_press_release
 
+urlpatterns = [
     # TODO: Is this used anymore? What is STATIC_GRAB?
-    url(r'^t/(?P<template>[^/]*)$', 'index'),
+    url(r'^t/(?P<template>[^/]*)$', index),
 
     # Semi-static views (these need to be rendered and have the login bar, but don't change)
-    url(r'^404$', 'render', {'template': '404.html'}, name="404"),
+    url(r'^404$', render, {'template': '404.html'}, name="404"),
```

Comment:
Not new to this PR, but it seems very odd to have a 404 page which returns an HTTP 200 code...especially when there's another view right below at the same URL which correctly returns a 404.